pkg/lightning/restore/check_info.go

```diff
@@ -82,27 +95,17 @@ func (rc *Controller) ClusterResource(ctx context.Context) error {
 	}
 	totalAvailable := typeutil.ByteSize(0)
 	for _, store := range result.Stores {
-		totalAvailable += store.Status.Available
+		totalAvailable += store.Status.Capacity
```

Comment:
better rename `totalAvailable` to `totalCapacity`? the message in line 111 may be inacurrate too.

Reply:
In acutally, I want to check the available rather than capacity. But when one lightning instance call this function,there may already import some data to TiKV cluster.